src/components/UserSelector.tsx

```diff
@@ -72,11 +68,9 @@ export const UserSelector: React.FC<Props> = ({
             <a
               key={user.id}
               href={`#user-${user.id}`}
-              onClick={() => {
-                onChange(user);
-              }}
+              onClick={() => selectAuthor(user)}
```

Comment:
The `href` attribute is used here for navigation, but it seems unnecessary since the `onClick` event is handling the user selection. Consider removing the `href` attribute to prevent any unintended navigation or page reload.